tests/functional/testplan/testing/multitest/test_xfail.py

```diff
@@ -57,13 +57,18 @@ def test_xfail():
     )
     result = plan.run()
 
-    assert result.report.passed is False
+    assert result.report.failed is True
```

Comment:
can we just `assert result.report.failed` please?

Reply:
sure